applications/FluidDynamicsApplication/tests/CylinderTest/cylinder_adjoint_parameters.json

```diff
@@ -0,0 +1,178 @@
+{
+    "problem_data"                     : {
+        "problem_name"    : "cylinder_2d",
+        "model_part_name" : "MainModelPart",
+        "domain_size"     : 2,
+        "start_step"      : 1.1,
+        "nsteps"          : 10,
+        "time_step"       :-0.1,
+        "echo_level"      : 1,
+        "parallel_type"   : "OpenMP"
+    },
+    "solver_settings"                  : {
+        "solver_type"                  : "adjoint_vmsmonolithic_solver",
+        "scheme_settings" : {
+            "scheme_type"              : "bossak",
+            "alpha_bossak"             :-0.3
+        },
+        "response_function_settings" : {
+            "response_type"            : "drag",
+            "sensitivity_model_part_name" : "NoSlip2D_cylinder",
+            "nodal_sensitivity_variables" : ["SHAPE_SENSITIVITY"],
+            "custom_settings" : {
+                "structure_model_part_name" : "NoSlip2D_cylinder",
+                "drag_direction"            : [1.0, 0.0, 0.0]
+            }
+        },
+        "dynamic_tau"                  : 1.0,
+        "oss_switch"                   : 0,
+        "echo_level"                   : 0,
+        "volume_model_part_name"       : "Parts_fluid",
+        "skin_parts"                   : ["NoSlip2D_cylinder"],
+        "no_skin_parts"                : ["VelocityConstraints2D_inlet","VelocityConstraints2D_walls"],
+        "linear_solver_settings"       : {
+            "solver_type"         : "AMGCL",
+            "verbosity"           : 0,
+            "coarse_enough"       : 500,
+            "tolerance"           : 1e-8
+        },
+        "model_import_settings"        : {
+            "input_type"     : "mdpa",
+            "input_filename" : "cylinder_2d"
+        },
+        "time_stepping"               : {
+            "automatic_time_step" : false,
+            "time_step"           : -0.1
```

Comment:
Does this time_step get used also?

Reply:
In fact, it is the other one that is not used (since time stepping is managed by the solver, not the main script). I've removed it from the test settings.